scripts/variantstore/wdl/extract/import_gvs.py

```diff
@@ -397,10 +393,6 @@ def convert_array_with_id_keys_to_dense_array(arr, ids, drop=[]):
 
         vd = vd.drop('allele_NO', 'allele_YES', 'allele_is_snp', 'allele_OK')
 
-        if unphase:
-            vd = vd.annotate_entries(LGT=vd.LGT.unphase())
-        vd = vd.select_entries(GT=hl.vds.lgt_to_gt(vd.LGT, vd.LA), **vd.entry)
```

Comment:
Confirming my understanding: we no longer want GT b/c it is too confusing once you start filtering samples (with `remove_dead_alleles=True`) b/c the alleles array changes and users don't know they need to change GT?

Reply:
we added GT because Lee didn't want users to have to do the extra work downstream and then we accidentally gave them a VDS where all the GTs were wrong. Time to just lets the users convert LGT!